SPECS/libnbd/libnbd.spec

```diff
@@ -33,18 +62,27 @@ BuildRequires:  glib2-devel
 
 # For bash-completion.
 BuildRequires:  bash-completion
+BuildRequires:  bash-completion-devel
 
+# Only for running the test suite.
 %if 0%{?with_check}
 BuildRequires:  coreutils
 BuildRequires:  gcc-c++
 BuildRequires:  gnutls-utils
 BuildRequires:  iproute
 BuildRequires:  jq
+%if !0%{?rhel}
 BuildRequires:  nbd
+%endif
 BuildRequires:  qemu-img
 BuildRequires:  util-linux
 %endif
 
+#%%if 0%%{?have_ocaml}
+# The OCaml runtime system does not provide this symbol
+#%%global __ocaml_requires_opts -x Stdlib__Callback
+#%%endif
+
```

Comment:
Why do we even need to keep the comments?

Reply:
Ah, I think I forgot to uncomment that. Let me dig into it again.

Reply:
Fixed